tests/test_mapping_service.py

```diff
@@ -362,3 +366,108 @@ def test_get_service_query(self):
     def test_post_service_query(self):
         """Test sparql generated by a service (that has values outside of where clause) with POST."""
         self.assert_post_sparql_results(self.client, SPARQL_FROM_SERVICE)
+
+
+
+# MAPPING_ENDPOINT = "https://bioregistry.io/sparql"
+MAPPING_ENDPOINT = "http://localhost:8000/sparql"
+BLAZEGRAPH_ENDPOINT = "http://localhost:9999/blazegraph/namespace/kb/sparql"
+
+FEDERATED_QUERY = f"""PREFIX owl: <http://www.w3.org/2002/07/owl#>
+SELECT DISTINCT ?o WHERE {{
+    SERVICE <{MAPPING_ENDPOINT}> {{
+        <http://purl.obolibrary.org/obo/CHEBI_24867> owl:sameAs ?o
+    }}
+}}"""
+
+# FEDERATED_QUERY = f"""PREFIX owl: <http://www.w3.org/2002/07/owl#>
+# SELECT DISTINCT ?o WHERE {{
+#     SERVICE <{MAPPING_ENDPOINT}> {{
+#         ?s ?p ?o
+#     }}
+# }}"""
+
+class TestFederatedSparql(unittest.TestCase):
```

Comment:
can you move this entire thing into `test_federated_sparql.py`? Then we can reuse some testing code and make it easier to understand